webportal/src/app/market_list/components/side_bar.jsx

```diff
@@ -64,35 +73,53 @@ const SideBar = () => {
       <SearchBox onSearch={searchKeyword} />
       <Text variant={'large'}>Types</Text>
       <Stack>
-        <TypeFilter
+        <FilterItem
           text='All'
-          selected={type === TYPE_ENUM.ALL}
-          onClick={changeType(TYPE_ENUM.ALL)}
+          selected={currentType === TYPE_ENUM.ALL}
+          onClick={changeUrl('type', TYPE_ENUM.ALL)}
         />
-        <TypeFilter
+        <FilterItem
```

Comment:
>FilterItem [](http://example.com/codeflow?start=9&length=10)

sorry for my ignorance, what's the difference between TypeFilter and FilterItem?

Reply:
In the file [type_filter.jsx](https://github.com/microsoft/openpaimarketplace/blob/0f6ada0797/webportal/src/app/market_list/components/type_filter.jsx) , the component is exported as FilterItem, so I think it should be aligned with the exported name here.
It is a missing alignment when changing name, and I also change the file name from `type_filter.jsx` to `filter_item.jsx` because the latter name is more suitable.